app/pubhub/helper.ts

```diff
@@ -1,3 +1,9 @@
 export const transformTimeToUtcString = (timeString: string) => {
   return timeString.split(" ").join("T")
 }
+
+export const regexDate = /^[0-9]{4}-[0-9]{2}-[0-9]{2}[ T][0-9]{2}:[0-9]{2}:[0-9]{2}[\.0-9A-Z]*$/
```

Comment:
A space sneaked in in front of the `T` in the middle. I don't know if it is a mistake?

Reply:
It's unfortunately not a mistake. I found out that sometimes we can also get a date in that format - no T but a space instead. 🙄